website/docs/Software/OpenBCISoftware/01-OpenBCI_GUI.md

```diff
@@ -288,26 +288,42 @@ Expert Mode Keyboard Shortcuts:
+First, go to Processing's website and [download the latest version of Processing](https://processing.org/download). For more information on Processing, you can review the official [Processing Getting Started Guide](https://processing.org/tutorials/gettingstarted) or the [Processing Documentation](https://processing.org/environment).
 
-On a Mac `Users/<user-name>/Documents/Processing`  
- On Windows `C:\Users\<user-name>\Documents\Processing`  
- On Linux `/home/<user-name>/sketchbook`
+Go ahead and move the Processing application to your `Applications` folder, and launch the application. If this is the first time that you are running Processing, it will create what it calls it's `Sketch` folder in the following locations:
 
-![GUI repo](../../assets/SoftwareImages/OpenBCISoftware/ganglion_GUI-repo.png)
+- On a Mac -`Users/<user-name>/Documents/Processing`  
+- On Windows - `C:\Users\<user-name>\Documents\Processing`  
+- On Linux -`/home/<user-name>/sketchbook`
 
-The OpenBCI GUI code repository is located on github [here](https://github.com/OpenBCI/OpenBCI_GUI). click on the `Clone or download` button in green on the right, and select `Download ZIP`. If you are a advanced GitHub user, go ahead and clone it, or fork it if you like.
+:::note
+Linux Users
+
+[Here is a walkthrough on how to install Processing on Ubuntu Linux.](http://www.artsnova.com/processing/installing-processing-ubuntu-linux-tutorial.html)
+:::
+
+:::note
+Mac Users
+
+Please use the __Intel 64-bit__ version of Processing 4 on all Mac computers. This will still run great on Apple Silicon. This allows us to continue using libraries that may not have a build for Arm64 architecture.
+:::
 
-If you downloaded the repository as a zip, please unzip it and you can change the folder name from `OpenBCI_GUI-master` to `OpenBCI_GUI`, if you prefer. You can then place this folder wherever you like (Mac Example: `Users/<user-name>/Documents/GitHub/OpenBCI_GUI`).
+### Downloading the code for the OpenBCI GUI
+
+The OpenBCI GUI code repository is located [here on Github](https://github.com/OpenBCI/OpenBCI_GUI). Complete the following steps to download it to your computer:
+
+- Click on the `Clone or download` button in green on the right, and select `Download ZIP`. If you are a advanced GitHub user, go ahead and clone it or fork it as your normally would.
+
+- If you downloaded the repository as a zip, please unzip it and you can change the folder name from `OpenBCI_GUI-master` to `OpenBCI_GUI`, if you prefer. You can then place this folder wherever you like (Mac Example: `Users/<user-name>/Documents/GitHub/OpenBCI_GUI`).
 
 :::info Copy 3rd Party Libraries
 Inside the `OpenBCI_GUI` folder, there is a folder called `libraries`. Theses are the 3rd party libraries that the OpenBCI GUI uses to work it's magic. You need to **copy** all of these folders into:
```

Comment:
Recommend removing "magic" from this sentence. "Theses" should also be "These"

Something like:
These are the third-party libraries that the OpenBCI GUI depends on to work.

Reply:
Agreed. If they were magic, we could just use .dll on every OS and every architecture. I'll ask for a dynamic library magic wand for Christmas.